packages/renderer/src/stores/navigation/navigation-registry-kubernetes.svelte.spec.ts

```diff
@@ -17,20 +17,28 @@
  ***********************************************************************/
 
 import type { KubernetesObject } from '@kubernetes/client-node';
-import { beforeEach, expect, test, vi } from 'vitest';
+import { afterEach, beforeEach, expect, test, vi } from 'vitest';
 
 import KubeIcon from '/@/lib/images/KubeIcon.svelte';
+import { NO_CURRENT_CONTEXT_ERROR } from '/@api/kubernetes-contexts-states';
 
-import { createNavigationKubernetesGroup } from './navigation-registry-kubernetes.svelte';
+import { createNavigationKubernetesGroup, unsubscribe } from './navigation-registry-kubernetes.svelte';
+
+const kubernetesRegisterGetCurrentContextResourcesMock = vi.fn();
+const kubernetesGetCurrentContextGeneralStateMock = vi.fn();
+
+(window as any).kubernetesRegisterGetCurrentContextResources = kubernetesRegisterGetCurrentContextResourcesMock;
```

Comment:
why not using `vi.mocked(window.kubernetesRegisterGetCurrentContextResourcesMock)` and inside the beforeEach do `(window as any).kubernetesRegisterGetCurrentContextResources = vi.fn()`

Reply:
to keep the same pattern used into this file (I just moved this line from inside the `beforeEach`.